csharp/src/Google.Protobuf/Properties/AssemblyInfo.cs

```diff
@@ -59,14 +59,18 @@
 
 [assembly: AssemblyVersion("2.4.1.555")]
 
+#if SIGNED
 [assembly: InternalsVisibleTo("Google.Protobuf.Test, PublicKey=" +

```

Comment:
It might be worth taking the approach of Noda Time here, making the public key a constant (conditionalized by the release). That way it makes the InternalsVisibleTo simpler. (That may be the cause of the build break at the moment - I haven't checked.)

See https://github.com/nodatime/nodatime/blob/master/src/NodaTime/Properties/AssemblyInfo.cs